src/index.njk

```diff
@@ -11,9 +11,9 @@ layout: oneColumnPage.njk
     <div class="govuk-grid-row">
         <h3 id="whats-new" class="govuk-heading-m">What’s new</h3>
 <ul class="govuk-list govuk-list--bullet">
+<li>2 December 2022: <a href="/patterns-hmrc-govuk-team/green-button/" class="govuk-link">Green button</a> guidance updated</li>
 <li>4 October 2022: Archived the <a href="/patterns-hmrc-govuk-team/anti-dumping-measures/" class="govuk-link">Anti-dumping duty measures</a> content pattern</li>
 <li>28 September 2022: Replaced the 'Videos, webinars and email alerts' content pattern with <a href="/patterns-hmrc-govuk-team/hmrc-email-updates-videos-webinars-community-forums/" class="govuk-link">HRMC email updates, video, webinars and community forums</a></li>
```

Comment:
I noticed on Friday that HMRC is spelt incorrectly on this item. Could you change it as part of this PR? Other than that, looks good.

Reply:
Good spot :) Done